packages/simcore-sdk/tests/unit/test_node_ports_common_file_io_utils.py

```diff
@@ -271,7 +272,7 @@ async def test_upload_file_to_presigned_links(
     assert effective_chunk_size <= used_chunk_size
     upload_links = await create_upload_links(num_links, used_chunk_size)
     assert len(upload_links.urls) == num_links
-    async with ProgressBarData(num_steps=1) as progress_bar:
+    async with ProgressBarData(num_steps=1, description=IDStr("")) as progress_bar:
```

Comment:
Why we insist in using `IDStr` for descriptions? :-)) ... Is there any rationale to  it?